src/vivarium_gates_mncnh/components/children.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Tuple
+
+import numpy as np
+import pandas as pd
+from vivarium import Component
+from vivarium.framework.engine import Builder
+
+from vivarium_gates_mncnh.constants import data_keys, data_values, models
+
+
+class NewChildren(Component):
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def sub_components(self) -> List[str]:
+        return [self.lbwsg]
+
+    def __init__(self):
+        super().__init__()
+        self.lbwsg = LBWSGDistribution()
+
+    def setup(self, builder: Builder):
+        self.randomness = builder.randomness.get_stream(self.name)
+        self.male_sex_percentage = data_values.INFANT_MALE_PERCENTAGES[
+            builder.data.load(data_keys.POPULATION.LOCATION)
+        ]
+
+    def empty(self, index: pd.Index) -> pd.DataFrame:
+        return pd.DataFrame(
+            {
+                "sex_of_child": data_values.PREGNANCY_OUTCOMES.INVALID_OUTCOME,
+                "birth_weight": np.nan,
+                "gestational_age": np.nan,
+            },
+            index=index,
+        )
+
+    def generate_children(self, index: pd.Index) -> pd.DataFrame:
+        sex_of_child = self.randomness.choice(
+            index,
+            choices=["Male", "Female"],
+            p=[self.male_sex_percentage, 1 - self.male_sex_percentage],
+            additional_key="sex_of_child",
+        )
+        lbwsg = self.lbwsg(sex_of_child)
+        return pd.DataFrame(
+            {
+                "sex_of_child": sex_of_child,
+                "birth_weight": lbwsg["birth_weight"],
+                "gestational_age": lbwsg["gestational_age"],
+            },
+            index=index,
+        )
+
+
+class LBWSGDistribution(Component):
```

Comment:
It's not possible to use or subclass the `LBWSGDistribution` component defined in VPH?

Reply:
I spent a little bit of time trying to get this to work but it seems like it is not worth it. This component only wants to sample the exposures and doesn't need the rest of the overhead from risk distributions.